tests/test_config.py

```diff
@@ -55,13 +55,13 @@ def test_create_destination_directory(self):
 
     @mock.patch('kinto.config.render_template')
     def test_hmac_secret_is_text(self, mocked_render_template):
-        config.init('kinto.ini', 'postgresql')
+        config.init('kinto.ini', 'postgresql', 'postgresql')
```

Comment:
We should use kwargs here to make it more readable:

```config.init('kinto.ini', backend='postgresql', cache_backend='postgresql')```

Should we also default cache_backend to backend if the parameter is None?

Reply:
quick question, in my understanding cache_backend cant really be none (it can be default, which would be memory) since we are asking for user input in a while loop in __main__.py. Please let me know if I am missing something here. 
Thanks

Reply:
There are two things, the CLI tool that use the config.init function and the config.init function that can be use from somewhere else and notably in the tests.

For the former, you are right, but one could use the config.init function and I find it better not to have to pass twice the same value.